### PR TITLE
[openexr] Update to 3.1.8

### PR DESCRIPTION
**Pull request by Osyotr**
https://github.com/AcademySoftwareFoundation/openexr/releases/tag/v3.1.8

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/openexr/fix-arm64-windows-build.patch [hunk 1]
```diff
@@ -0,0 +1,23 @@
+From 46af6b978aa5a5e55066776b0adead25247c8829 Mon Sep 17 00:00:00 2001
+From: Christoph Gohlke <cgohlke@cgohlke.com>
+Date: Wed, 7 Jun 2023 20:18:48 -0700
+Subject: [PATCH] Fix build error on win_arm64
+
+Signed-off-by: Christoph Gohlke <cgohlke@cgohlke.com>
+---
+ src/lib/OpenEXRCore/internal_dwa_simd.h | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
```
Comment (Adela0814) [inline ports/openexr/fix-arm64-windows-build.patch:10]: Please remove these lines, thanks.

**Review by Adela0814 — CHANGES_REQUESTED** [review]

**Comment by Adela0814** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". That way, I can be aware that you've responded since you can't modify the tags.

File: ports/openexr/fix-arm64-windows-build.patch (hunk 1 above)
Comment (Osyotr) [reply ports/openexr/fix-arm64-windows-build.patch:10]: I just downloaded this patch from upstream repo. Should I bother in such cases?

Comment (Adela0814) [reply ports/openexr/fix-arm64-windows-build.patch:10]: Could you please remove these unnecessary lines manually. Thank you very much.

**Review by Adela0814 — CHANGES_REQUESTED** [review]

**Review by Adela0814 — APPROVED** [review]

**Pull request merged**